src/roadmap/kessel.py

```
@@ -102,13 +104,11 @@ def subject_from_identity(identity: dict[str, t.Any], domain: str) -> t.Any:
     return principal_subject(str(user_id), domain)
 
 
-def host_groups_for(client: t.Any, subject: t.Any) -> list[str]:
+async def host_groups_for(client: t.Any, subject: t.Any) -> list[str]:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `client`

(ANN401)